src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayCost = 40;
+  const shortTerm = 3;
+  const longTerm = 7;
+  const shortTermDiscount = 20;
+  const longTermDiscount = 50;
+
+  if (days < shortTerm) {
+    return days * oneDayCost;
+  }
+
+  if (days < longTerm) {
+    return days * oneDayCost - shortTermDiscount;
```

Comment:
The condition `days < longTerm` should be `days >= shortTerm && days < longTerm` to ensure that the short-term discount is only applied when the number of days is between 3 and 6 inclusive. Currently, the condition does not check if the days are greater than or equal to the short-term threshold.